src/store/reducers/followedAccounts.js

```diff
@@ -0,0 +1,44 @@
+import { getIndexOfFollowedAccount } from '../../utils/followedAccounts';
+import actionTypes from '../../constants/actions';
+
+const followedAccounts = (state = { accounts: [] }, action) => {
+  const accounts = [...state.accounts];
+  let indexOfAccount;
+  let changedAccount;
+
+  switch (action.type) {
+    case actionTypes.followedAccountsRetrieved:
+      return { accounts: Array.isArray(action.data) ? action.data : [action.data] };
+
+    case actionTypes.followedAccountAdded:
+      accounts.push(action.data);
```

Comment:
I was thinking about whether this should check if the account is already on the list, but I guess it is good as long as the "edit" component doesn't allow the same account twice. Then we would have a problem with using `publicKey` as a unique identifier.